### PR TITLE
Update docs

### LINKED ISSUE #54: [DOCS] <Improve the documentation>
### 

I think the doc can be improve in many ways. 
- Use semantic ordered list for the instructions, instead of `#1` or `#2`
- Remove the comments which are in parentheses. As, if you click on the copy button, they also get copied. Bad UX.
- Add steps like `cd <project-directory>` to remove any confusion

For reference here's how tailwindcss.com does it.

image

It's simple & to the point. Any feedback is welcome.

### PR DESCRIPTION
**Pull request by subhoghoshX**
### 🛠️ Fixes Issue

Closes #54 

### 👨‍💻 Changes proposed

Update the steps to run the project locally. Also make the it look cleaner.

### ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,60 +48,66 @@ feat: added authentication using supabase auth
...
 
-`#1` Make sure you have installed [npm](https://docs.npmjs.com/cli/init) in your local machine where you want to start app
+The repository has a `.vscode` folder that contains `settings.json` and `extensions.json`. The `settings.json` file configures your VS Code editor to use `eslint` and `prettier` on every code save action (`ctrl + s`). The `extension.json` file contains a list of VS Code extensions, VS Code will show these extensions as suggestions in the extensions tab. After installing these extensions, auto linting and formatting should start working.
 
-`#2` Run `npm install` to install all the required packages
+### Steps:
 
-`#3` The repository has a `.vscode` folder that contains `settings.json` and `extensions.json`. The `settings.json` file configures your VS Code editor to use `eslint` and `prettier` on every code save action (`ctrl + s`). The `extension.json` file contains a list of VS Code extensions, VS Code will show these extensions as suggestions in the extensions tab. After installing these extensions, auto linting and formatting should start working.
+1. Clone this repo
```
Comment (iShibi) [inline README.md:59]:
```suggestion
1. Clone this repo

```

File: README.md [hunk 2]
```diff
@@ -48,60 +48,66 @@ feat: added authentication using supabase auth
...
+   ```sh
+   git clone https://github.com/ykdojo/defaang.git
+   ```
+2. Go into the project root directory
```
Comment (iShibi) [inline README.md:63]:
```suggestion

2. Go into the project root directory
```

**Review by iShibi — CHANGES_REQUESTED** [review]
Add a new line before and after a command example, just like the other steps

File: README.md (hunk 2 above)
Comment (subhoghoshX) [reply README.md:63]: Prettier formatted it automatically. I think prettier check will fail if I remove it.

**Comment by subhoghoshX** [comment]
> Add a new line before and after a command example, just like the other steps

Prettier formatted it automatically. I think prettier check will fail if I remove it.

File: README.md (hunk 2 above)
Comment (iShibi) [reply README.md:63]: I ran `npm run format:fix` on this and it seems to add a newline before and after correctly. Can you run this cmd and see it fixes the formatting for you?

**Comment by iShibi** [comment]
@subhoghoshX I ran `npm run format:fix` on this and it seems to add a newline before and after correctly. Can you run this cmd and see it fixes the formatting for you?

**Comment by subhoghoshX** [comment]
> @subhoghoshX I ran `npm run format:fix` on this and it seems to add a newline before and after correctly. Can you run this cmd and see it fixes the formatting for you?

Yeah done. Added it manually. I'm not sure why my prettier acted weirdly back then. Thanks btw.

**Comment by iShibi** [comment]
You're welcome. You can mark the above reviews as resolved now.

**Review by rohitdasu — APPROVED** [review]

**Comment by ykdojo** [comment]
feel free to merge it @rohitdasu @subhoghoshX

**Comment by rohitdasu** [comment]
@subhoghoshX are you done with the changes?

**Comment by subhoghoshX** [comment]
Yes. You can merge it. Thanks.

**Pull request merged**